internal/execution/executor_test.go

```diff
@@ -53,11 +55,11 @@ func TestBasics(t *testing.T) {
 				Object: map[string]any{
 					"apiVersion": "v1",
 					"kind":       "ConfigMap",
-					"metadata": map[string]string{
+					"metadata": map[string]any{
```

Comment:
Need to pass metadata as type map[string]any or NestedString will throw the error